packages/devextreme-monorepo-tools/package.json

```diff
@@ -1,4 +1,7 @@
 {
   "name": "devextreme-monorepo-tools",
-  "version": "24.1.2"
+  "version": "24.1.2",
+  "devDependencies": {
+    "ts-jest": "^29.1.2"
```

Comment:
`devextreme-monorepo-tools` were able to use `ts-jest` because it was hoisted to the top level by npm, when being declared as a dependency in other workspaces. After update, it is no longer hoisted. Neither are `jest` and `@types/jest`.

I manually elevated `jest` and `@types/jest` to the top level, because`@nx/jest` needs them there, and referenced `ts-jest` here.